sdk/batch/Azure.ResourceManager.Batch/api/Azure.ResourceManager.Batch.netstandard2.0.cs

```diff
@@ -1638,12 +1795,161 @@ public enum InterNodeCommunicationState
         Enabled = 0,
         Disabled = 1,
     }
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public readonly partial struct IssueType : System.IEquatable<Azure.ResourceManager.Batch.Models.IssueType>
```

Comment:
The name is too generic.

Reply:
updated in new PR https://github.com/Azure/azure-sdk-for-net/pull/45847